test/unit/reductions/test_smoke.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+from lightgbm import LGBMClassifier
+import numpy as np
+import pandas as pd
+import pytest
+from sklearn.linear_model import LogisticRegression
+from sklearn.svm import SVC
+from sklearn.tree import DecisionTreeClassifier
+
+from fairlearn.reductions import ExponentiatedGradient, GridSearch, DemographicParity, \
+    EqualizedOdds
+
+
+@pytest.mark.parametrize("Mitigator", [ExponentiatedGradient, GridSearch])
+@pytest.mark.parametrize("Constraints", [DemographicParity, EqualizedOdds])
+@pytest.mark.parametrize("Estimator", [LGBMClassifier, LogisticRegression, SVC,
+                                       DecisionTreeClassifier])
+def test_smoke(Mitigator, Constraints, Estimator):
```

Comment:
Thanks!

Reply:
MacOS of course doesn't just work with the pip install lightgbm, so I've excluded that case there. This specific behavior shouldn't be different on different OSes, so I think it's fine. I think at some point we'll have to expand this to other types (pytorch, tensorflow, xgboost, etc.) as well, so I'll leave the installation fixes for then....